doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,109 @@
+
+New entry creation
+==================
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following paths:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
+* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.
+
+.. note::
+   A *design* is a specific configuration of hardware or software implementation of adopted Bluetooth specifications.
+
+
+Existing entry update
+=====================
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You changed an existing product's design significantly so that the testing is required.
+* Your product can be added as a new model to one of your existing qualifications.
+* You switched to another SDK version for an existing product.
+
+If you want to switch to another SDK version for an existing product, perform the following steps:
+
+1. Go to the `Nordic Semiconductor Infocenter`_, search for the SoC or SiP model you use, and navigate to the relevant Compatibility Matrix directory.
+
+#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:
+
+  a. That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.
+     If yes, perform one of the following actions:
+
+     * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones currently used for your product, then no further action is required.
+     * If the version you want to use has at least one QDID number that is different from the numbers already used for your product, then a new qualification is required.
+       Log into the `Launch Studio`_ tool and follow the steps for `Qualification process with no required testing`_ option.
+
+  b. That the version you want to use is compatible with versions of other Subsystems you want to keep for your product.
+
+.. note::
+
+   An existing entry can be edited only by the person that completed the listing.
+   If you want to change the entry owner, contact SIG support.
+
+
+Qualification listing
+*********************
+
+`Bluetooth Qualification Listing`_ search lists all qualified designs. Use it in the following cases:
+
+* To check if your qualification process succeeded.
+* To search for existing designs that can be used in your product.
+* To check if you can add your device model to an existing qualification entry.
+
+
+Dependencies
```

Comment:
```suggestion
Matter certification by inheritance
```
Maybe let's reference Matter in the title? If not, can we change the header to something else? "Dependencies" can suggest both things that depend on the certification and things that the certification depends on, so I'd rather be more specific.